rmf_demos_tasks/rmf_demos_tasks/dispatch_teleop.py

```diff
@@ -93,7 +93,10 @@ def __init__(self, argv=sys.argv):
 
         # Define task request category
         request["category"] = "compose"
-
+        
+        if self.args.robot is None and self.args.fleet:
```

Comment:
This can be simplified to 
```suggestion
        if self.args.fleet:
```

Since all of this is within the `else` logic.

Similarly in other files.

Reply:
@Yadunund 
Done. Please review